client/src/components/dialog/Dialog.css

```diff
@@ -0,0 +1,17 @@
+.dialog-overlay {
+  position: fixed;
+  inset: 0px;
+  backdrop-filter: blur(0.2rem);
```

Comment:
Adds a nice little blur to the backdrop. You can see here: 

![image](https://github.com/rohanshiva/gitgame/assets/20916697/c58d499f-5403-4998-ba75-41b17de4af22)

Reply:
I really like this!

Reply:
Hmm, I made the shadow pop a bit more. I think shadow on top will look a bit weird. I made it so that it has an effect where the shadow is not the same everywhere. 
![image](https://github.com/rohanshiva/gitgame/assets/20916697/b4ad12b2-b567-4765-9995-649b395dad05)

Let me know how this looks.

Reply:
Looks good!